SDK/AppCenter/Microsoft.AppCenter.Windows.Shared/Ingestion/Http/HttpNetworkAdapter.cs

```diff
@@ -125,6 +137,10 @@ private async Task<HttpResponseMessage> SendRequestAsync(HttpRequestMessage requ
             {
                 return await HttpClient.SendAsync(request, cancellationToken).ConfigureAwait(false);
             }
+            catch (HttpRequestException e)
```

Comment:
Are you sure that we should catch all these exception without additional conditions?